src/main/java/com/project/model/Application.java

```diff
@@ -21,53 +22,53 @@
 
     @NotBlank(message = "Vardas negali būti tuščias")
     @Size(max = 255, message = "Vardas negali būti ilgesnis nei 255 simboliai")
-    @Column(name="full_name")
+    @Column(name = "full_name")
     private String name;
 
     @NotBlank(message = "Telefono numeris negali būti tuščias")
     @Size(max = 20, message = "Telefono numeris negali būti ilgesnis nei 20 simbolių")
-    @Column(name="phone_number")
+    @Column(name = "phone_number")
     private String phone_number;
 
     @NotBlank(message = "Mokymosi įstaiga negali būti tuščia")
     @Size(max = 255, message = "Mokymosi įstaigos pavadinimas negali būti ilgesnis nei 255 simboliai")
-    @Column(name="education")
+    @Column(name = "education")
     private String education;
 
     @NotBlank(message = "Pomėgių laukas negali būti tuščias")
     @Size(max = 1500, message = "Pomėgių lauke negali būti daugiau nei 1500 simbolių")
-    @Column(name="free_time")
+    @Column(name = "free_time")
     private String free_time;
 
     @NotNull(message = "Ar reikalinga trišalė sutartis privalo būti pasirinkta")
-    @Column(name="agreement")
+    @Column(name = "agreement")
     private boolean agreement;
 
     @Size(max = 1500, message = "Komentaras negali būti ilgesnis nei 1500 simboliai")
-    @Column(name="comment")
+    @Column(name = "comment")
     private String comment;
 
     @NotNull(message = "Ar tinka akademijos laikas privalo būti pasirinkta")
-    @Column(name="academy_time")
+    @Column(name = "academy_time")
     private boolean academy_time;
 
     @NotBlank(message = "Dalyvavimo akademijoje priežastis negali būti tuščia")
     @Size(max = 255, message = "Dalyvavimo akademijoje priežastis negali būti ilgesnė nei 1500 simboliai")
-    @Column(name="reason")
+    @Column(name = "reason")
     private String reason;
 
     @NotBlank(message = "Technologijų ir patirties laukas negali būti tuščias")
     @Size(max = 1500, message = "Technologijų ir patirties lauke negali būti daugiau nei 1500 simbolių")
-    @Column(name="technologies")
+    @Column(name = "technologies")
     private String technologies;
 
     @NotBlank(message = "Iš kur sužinojai apie akademiją negali būti tuščias")
     @Size(max = 255, message = "Iš kur sužinojai apie akademiją negali būti ilgesnis nei 255 simboliai")
-    @Column(name="source")
+    @Column(name = "source")
     private String source;
 
     @NotBlank(message = "Paraiškos data negali būti tuščia")
-    @Column(name="application_date")
+    @Column(name = "application_date")
     private String application_date;
```

Comment:
As mentioned above. Don't use underscore when naming your variables. Change it to something like `applicationDate`.

Reply:
changed several fields' names in Application class to better reflect what values are held in them